.goreleaser.yml

```diff
@@ -0,0 +1,102 @@
+## Bitnami Sealed Secrets
+
+release:
+  # Comment out to enable publishing
+  disable: true
+
+builds:
+- binary: controller
+  main: ./cmd/controller/
+  env:
+  - CGO_ENABLED=0
+  goarch:
+  - amd64
+  - arm
+  - arm64
+  goos:
+  - linux
+  - darwin
+  - windows
+  ldflags: -X main.VERSION={{.Version}}
+- binary: kubeseal
+  main: ./cmd/kubeseal/
+  env:
+  - CGO_ENABLED=0
+  goarch:
+  - amd64
+  - arm
+  - arm64
+  goos:
+  - linux
+  - darwin
+  - windows
+  ldflags: -X main.VERSION={{.Version}}
+
+brew:
+  github:
+    owner: bitnami-labs
+    name: sealed-secrets
+  homepage: "https://github.com/bitnami-labs/sealed-secrets/"
+  description: "A Kubernetes controller and tool for one-way encrypted Secrets"
+  skip_upload: true
+  # Replace with existing brew formula
+  test: |
+    system "#{bin}/program --version"
+
+dockers:
+- goarch: amd64
+  goos: linux
+  binaries:
+  - controller
+  dockerfile: ./docker/Dockerfile
+  image_templates:
+  - bitnami-labs/sealed-secrets-controller:latest
+  - bitnami-labs/sealed-secrets-controller:amd64-latest
+  - bitnami-labs/sealed-secrets-controller:amd64-{{ .Version }}
+  # Comment below out to push to configured repository
+  skip_push: true
+- goarch: arm64
+  goos: linux
+  binaries:
+  - controller
+  dockerfile: ./docker/Dockerfile
+  image_templates:
+  - bitnami-labs/sealed-secrets-controller:arm64-latest
+  - bitnami-labs/sealed-secrets-controller:arm64-{{ .Version }}
+  # Comment below out to push to configured repository
+  skip_push: true
+- goarch: arm
+  goarm: 6
+  goos: linux
+  binaries:
+  - controller
+  dockerfile: ./docker/Dockerfile
+  image_templates:
+  - bitnami-labs/sealed-secrets-controller:armv6-latest
+  - bitnami-labs/sealed-secrets-controller:armv6-{{ .Version }}
+  # Comment below out to push to configured repository
+  skip_push: true
+
+archive:
```

Comment:
It would seem GoReleaser doesn't (yet?) support selecting which binaries from the `builds:` section get included in archives.
There's a ticket about it here: https://github.com/goreleaser/goreleaser/issues/705
Seems like a fairly major arch change in GoReleaser required.

On this topic, are you happy with the default inclusion of README and LICENSE files? 
We can exclude these if you prefer.

[GoReleaser Archive docs](https://goreleaser.com/customization/#Archive)

Reply:
> On this topic, are you happy with the default inclusion of README and LICENSE files?
We can exclude these if you prefer.

Yep, including those files is fine and appropriate.